src/components/NavBar/SuggestionRow.tsx

```diff
@@ -167,21 +171,16 @@ export const TokenRow = ({ token, isHovered, setHoveredIndex, toggleOpen, index,
       style={{ background: isHovered ? vars.color.lightGrayOverlay : 'none' }}
     >
       <Row style={{ width: '65%' }}>
-        {!brokenImage && token.logoURI ? (
-          <LogoContainer>
-            <Box
-              as="img"
-              src={token.logoURI.includes('ipfs://') ? uriToHttp(token.logoURI)[0] : token.logoURI}
-              alt={token.name}
-              className={clsx(loaded ? styles.suggestionImage : styles.imageHolder)}
-              onError={() => setBrokenImage(true)}
-              onLoad={() => setLoaded(true)}
-            />
-            <L2NetworkLogo networkUrl={L2Icon} size="16px" />
-          </LogoContainer>
-        ) : (
-          <Box className={styles.imageHolder} />
-        )}
+        <StyledLogoContainer>
```

Comment:
I think this is cleaner to do in an inline style property (which is allowed by your AssetLogo component

    style={{marginRight: 8}}

Reply:
That would shift the L2 icon over as well. I do plan to consolidate L2 icon rendering into AssetLogo in a future PR to remove the duplicated logic, so once that's done I can add the inline styling then. I'll add a TODO